examples/vercel-ai-chatbot/auth.ts

```diff
@@ -10,30 +10,39 @@ declare module 'next-auth' {
   }
 }
 
-export const {
-  handlers: { GET, POST },
-  auth
-} = NextAuth({
-  providers: [GitHub],
-  callbacks: {
-    jwt({ token, profile }) {
-      if (profile) {
-        token.id = profile.id
-        token.image = profile.avatar_url || profile.picture
-      }
-      return token
-    },
-    session: ({ session, token }) => {
-      if (session?.user && token?.id) {
-        session.user.id = String(token.id)
-      }
-      return session
-    },
-    authorized({ auth }) {
-      return !!auth?.user // this ensures there is a logged in user for -every- request
+// Didn't want to setup a Github App
```

Comment:
Github app setup is done now. You can log into vercel using devops@vellum.ai and the env vars are configured correctly to point to the Oauth app I created for this.